native/cocos/bindings/manual/jsb_global_init.cpp

```diff
@@ -220,6 +220,7 @@ void jsb_init_file_operation_delegate() { //NOLINT
 }
 
 bool jsb_enable_debugger(const ccstd::string &debuggerServerAddr, uint32_t port, bool isWaitForConnect) { //NOLINT
+#if SCRIPT_ENGINE_TYPE == SCRIPT_ENGINE_V8
```

Comment:
OpenHarmony supports debugger?

Reply:
not support network debug， can debugger js on ide。

Reply:
Got it.